src/main/java/net/dv8tion/jda/api/utils/FutureUtil.java

```diff
@@ -0,0 +1,40 @@
+package net.dv8tion.jda.api.utils;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.Function;
+
+public class FutureUtil
```

Comment:
If you put this in the API package it has to be documented, might be something we can only put in internal.